continuous-pipe.yml.erb

```diff
@@ -54,6 +57,7 @@ tasks:
       services:
         web:
           image: quay.io/organisation/<%= config.name %>
```

Comment:
Shouldn't `organisation` be part of the `config` ?

Reply:
it's not something Hem currently asks for

Reply:
seedinit.rb can do it though